app/models/concerns/sufia/solr_document_behavior.rb

```diff
@@ -7,6 +7,14 @@ module SolrDocumentBehavior
     include Sufia::SolrDocument::Export
     include Sufia::SolrDocument::Characterization
 
+    def all_titles
```

Comment:
we shoud not do this. The way to do this is to fix the 29 broken tests.  If you need help, please get as far as you can an then push the branch with broken tests.